docs/modules/ROOT/pages/troubleshooting/index.adoc

```diff
@@ -149,3 +149,23 @@ kubectl get secrets -o json |
         {auths: .}
     '
 ----
+
+
+== Bundle properties are not permitted in a FBC fragment for OCP version
```

Comment:
I assume there will be a way to directly link to this header from our error message?

Related: is there a way to avoid (or at least reduce the chances of accidentally breaking that link (e.g. if we change the header text)?